### DISCUSSION TITLE
Yolov8 segmentation head

### DISCUSSION THREAD

**Discussion by AwayFromKeyb0ard** [ultralytics/ultralytics#13823] [Q&A]
i saw in another discussion here in github that yolo v8 segmentation model uses the segmentation architecture that yolact have. Is that correct? I am asking because i want to describe yolo v8 segmentation architecture in a master thesis. Also, i noticed that it uses protonet and coefficients that is used in yolact too.

**Comment by glenn-jocher** [comment]
@N0lanakis hello!

Thank you for your question and for considering YOLOv8 for your master thesis. Yes, you are correct that YOLOv8's segmentation head shares similarities with the YOLACT architecture. Specifically, YOLOv8 employs a Protonet and coefficient-based approach for instance segmentation, which is inspired by YOLACT.

For a more detailed understanding, YOLOv8's segmentation head generates a set of prototype masks and corresponding mask coefficients for each detected object. This allows the model to produce high-quality instance segmentation masks efficiently.

If you need further technical details or have specific questions about the implementation, feel free to ask! You can also refer to our [documentation on segmentation](https://docs.ultralytics.com/tasks/segment/) for more insights.

Best of luck with your thesis! 😊
